docs-source/spring/content/development/cli/_index.md

```diff
@@ -1,16 +1,10 @@
 ---
-title: Using the CLI
+title: Using the Oracle Spring CLI
```

Comment:
you cannot call it "Oracle Spring CLI" - you need to use the correct product name, e.g. "CLI for Oracle Backend as a Service for Spring Cloud" 
i think the title "using the CLI" was fine as it was

Reply:
`CLI for Oracle Backend as a Service for Spring Cloud` is good enough to a description but not for a title or menu and only CLI is not meaningful. OBaaS CLI?

Reply:
"Using the CLI" is fine - it is obvious from the context what we are talking about